src/pages/en/about/security-notice.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Security Notice
+description: If you find a vulnerability in one of our project, please report it using the procedure detailed here
+layout: /src/layouts/Page.astro
+---
+
+This is the security notice for all QuiltMC repositories. The notice explains how vulnerabilities should be reported. We also support the [security.txt](/.well-known/security.txt) standard.
+
+# Reporting a Vulnerability
+
+If you've found a vulnerability, we would like to know so we can fix it before it is released publicly. **Do not open a GitHub issue for a found vulnerability.**
```

Comment:
I've changed the tone here to be more instructional, i.e "This is what you must do" instead of "this is what we want the end result to be".

```suggestion
If you've found a vulnerability, please let us know privately so that we can fix it before it is released publicly. **Do not open a GitHub issue to report a security vulnerability.**
```